normandy/recipes/api/serializers.py

```diff
@@ -21,10 +24,137 @@ class Meta:
         ]
 
 
+class ApprovalSerializer(serializers.ModelSerializer):
+    creator = UserSerializer(read_only=True)
+    creator_id = serializers.IntegerField(source='creator.id', write_only=True)
```

Comment:
I think you can use `PrimaryKeyRelatedField` here, which may allow you to drop the three other methods on this class. I'm not 100% on this but the validation and extra logic for `create` and `update` seem like things DRF should handle for us already.